src/vs/workbench/contrib/chat/common/chatSessionsService.ts

```diff
@@ -14,6 +14,7 @@ import { IChatAgentRequest } from './chatAgents.js';
 import { IRelaxedExtensionDescription } from '../../../../platform/extensions/common/extensions.js';
 
 export interface IChatSessionsExtensionPoint {
+	readonly id: string;
```

Comment:
I think that we should only have `type` now. It was previously called `id` but I just changed this and having both will be confusing